web/src/dialog_widget.ts

```diff
@@ -174,6 +175,7 @@ export function launch(conf: DialogWidgetConfig): string {
     // * always_visible_scrollbar: Whether the scrollbar is always visible if modal body
     //   has scrollable content. Default behaviour is to hide the scrollbar when it is
     //   not in use.
+    // * ignore_overlay_click: Whether clicking on the overlay will not close the modal.
```

Comment:
I think we want a name that's more explicit about it closing the modal. Perhaps ideal would be `close_on_overlay_click`, and having it default to true? I'm also not sure if it's accurate to call it `close_on_background_click`, but that would be even more intuitive.